python/tvm/contrib/pytorch.py

```diff
@@ -0,0 +1,57 @@
+"""PyTorch bridge wrapping torch tensor."""
+
+import ctypes
+from decorator import decorate
+from .. import register_extension, TypeCode
+
+def to_pytorch(module):
+    """Wrap a TVM function as PyTorch function
+
+    Parameters
+    ----------
+    func : Function
+        A TVM function that can take positional arguments
+
+    args : Tuple of arguments
+        Tuple of arguments to pass to the wrapped function
+
+    Returns
+    -------
+    func : Function
+        A function that can take PyTorch tensor as argument
+        in places that used to expect TVM NDArray.
+    """
+    #import pytorch, check for pytorch tensor
+    import torch
+    def converter(func, *args):
+        args = tuple([FireTensor(arg) if isinstance(arg, torch.Tensor) else arg for arg in args])
+        return func(*args)
+
+    def wrapper(*args):
+        module(*args)
+
+    return decorate(wrapper, converter)
+
+@register_extension
+class FireTensor(object):
+    """Class to wrap PyTorh tensor"""
+    _tvm_tcode = TypeCode.ARRAY_HANDLE
+
+    def __init__(self, tensor):
```

Comment:
we can make the constructor to take in the handle, so it can be reused for other DLManagedTensor objects